src/cli/opts.rs

```diff
@@ -99,7 +99,15 @@ pub enum Command {
 
     /// Lists saved checkpoints of the swaps
     #[clap(aliases = &["lc"])]
-    ListCheckpoints,
+    ListCheckpoints {
+        #[clap(
+            short,
+            long,
+            default_value = "all",
+            possible_values = &["all", "All", "available", "Available", "available_for_restore"],
```

Comment:
`available-for-restore` seems more common in cli context

Reply:
ed1fbb0190dc44982d9c0829830121df88294dd2